webapp/src/modules/archivedBid/reducer.js

```diff
@@ -0,0 +1,30 @@
+import { ARCHIVE_BID, UNARCHIVE_BID } from './actions'
+
+const INITIAL_STATE = {
+  data: {}
+}
+
+export function archivedBidReducer(state = INITIAL_STATE, action) {
+  switch (action.type) {
+    case ARCHIVE_BID: {
+      const { bid } = action
+      return {
+        data: {
+          ...state.data,
+          [bid.id]: true
+        }
+      }
+    }
+    case UNARCHIVE_BID: {
+      const { bid } = action
+      return {
+        data: {
+          ...state.data,
+          [bid.id]: false
```

Comment:
it might be useful, instead of storing true/false, having timestamp/undefined. So:

```
data: {
  [bid.id]: Date.now()
````

and

```
data: {
  ...utils.omit(state.data, [bid.id])
```

if you don't find the timestamp useful, I still think cleaning the state might be worth it